patterns/templates/jolla-hw-adaptation-@DEVICE@.yaml

```diff
@@ -57,15 +56,6 @@ Requires:
 # enable device lock and allow to select untrusted software
 - jolla-devicelock-plugin-encsfa
 
-# Settings app needs this since update15
-- jolla-settings-layout
-
-# For devices with cellular modem. Those without one, please comment out:
-- pattern:jolla-sailfish-cellular-apps
-- telepathy-ring
-# For multi-SIM devices
-#- jolla-settings-networking-multisim
-
```

Comment:
geoclue-provider-mlsdb and csd below should be in configuration instead of hw-adaptation-*